amundsen-kube-helm/templates/helm/values.yaml

```diff
@@ -30,6 +30,43 @@ tolerations: []
 # podAnnotations -- amundsen application wide configuration of podAnnotations. This applies to search, metadata, frontend and neo4j. Elasticsearch has it's own configuation properties for this. [ref](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
 podAnnotations: {}
 
+# Optional Flask App wrapper to wrap your Amundsen Microservices
+flaskApp:
+  #  flaskApp.module -- 'flaskoidc' if you want to enable OIDC
+  module: ""
+  #  flaskApp.class -- 'FlaskOIDC' if you want to enable OIDC
+  class: ""
+
+# To enable OIDC,
+#     - please make sure to fill in the mandatory variables under 'envVars' for each service.
+#     - please make sure set frontend.config.class: amundsen_application.oidc_config.OidcConfig
+# Everything under oidc.configs will be parsed as environment variables for each service.
+# Following variables must be set under `envVars` for each service.
+# - FLASK_OIDC_CLIENT_ID
+# - FLASK_OIDC_CLIENT_SECRET
```

Comment:
would it make sense to pull client ID and/or secret from k8s secrets? otherwise users will need to provide sensitive values in values.yaml which might be tricky if you are trying to store your chart/deployment in git

something like:
```
containers:
  - name: frontend
    envFrom:
      - secretRef:
          name: frontend-oidc-client-secret
```

Reply:
excuse my lack of knowledge of k8s, but I am not sure how that would work with the pipelines. If we are injecting these variables via CI/CD or GitHub actions, would we still be able to create those secrets in K8s containers?

Reply:
I think this is more on the case-to-case scenario. For example, we used the Github secrets at one time to inject these values for one of our instances. thoughts?

Reply:
indeed it's a case to case scenario, we can leave it be for now